example-app/src/main/java/com/hcaptcha/example/MainActivity.java

```diff
@@ -121,6 +123,14 @@ public void onClickReset(final View view) {
         hCaptcha = null;
     }
 
+    public void onClickDestroy(final View view) {
+        if (hCaptcha != null) {
+            hCaptcha.destroy();
+        }
```

Comment:
If this doesn't exist don't you just want to end the call here? hCaptcha would already be null, maybe setTokenTextView is not needed either?